cmd/ignite/cmd/root.go

```diff
@@ -52,6 +58,38 @@ func NewIgniteCommand(in io.Reader, out, err io.Writer) *cobra.Command {
 			// Create the directories needed for running
 			util.GenericCheckErr(util.CreateDirectories())
 
+			var configFilePath string
+
+			// If an ignite config flag is set, use it as the config file, else check
+			// if the global config file exists.
+			// If a config file path is passed, configure ignite using it.
+			if configPath != "" {
+				configFilePath = configPath
+			} else {
+				// Check the default config location.
+				if _, err := os.Stat(constants.IGNITE_CONFIG_FILE); !os.IsNotExist(err) {
+					log.Infof("Found default ignite configuration file %s", constants.IGNITE_CONFIG_FILE)
+					configFilePath = constants.IGNITE_CONFIG_FILE
+				}
+			}
+
+			if configFilePath != "" {
+				log.Infof("Using ignite configuration file %s", configFilePath)
```

Comment:
We decided to keep these logs:
- increase the log level so it's not noisy in Info
- also log when there is no config found at all